### PR TITLE
added SPARK_HOME path the the connector settings in AS Spark Python notebook

### PR DESCRIPTION
**Pull request by mbelsare**

### CODE REVIEW

File: notebooks/spark/AerospikeSparkPython.ipynb [hunk 1]
```diff
@@ -1250,7 +1250,7 @@
     "#user specified index \"aerospike.sindex\"\n",
     "siNameDF = spark.read.format(\"aerospike\").schema(si_schema)\\\n",
     ".option(\"aerospike.set\", si_set)\\\n",
-    ".option(\"aerospike.partition.factor\",1)\\\n",
+    ".option(\"aerospike.partition.factor\",1)\\\n",g
```
Comment (rahul-aerospike) [inline notebooks/spark/AerospikeSparkPython.ipynb:1253]: `g` seems to a typo?

Comment (mbelsare) [reply notebooks/spark/AerospikeSparkPython.ipynb]: fixed it.

**Review by rahul-aerospike — APPROVED** [review]
LGTM, assuming the notebook ran fine with binder.

**Comment by neelp-git** [comment]
LGTM as long as it runs in binder and downloaded container.

**Review by artanderson — APPROVED** [review]
AerospikeSparkPython.ipynb runs fine, but the AerospikeSparkScala.ipynb is not able to run due to not having a spylon kernel installed. I'm not sure this notebook has ever run in binder or the container though. Otherwise, builds fine.

**Comment by mbelsare** [comment]
@artanderson well I dont think the scala notebook was run before. Looks like the spylon_kernel needs to be installed on the host machine where the virtual env comes up

**Pull request merged**